sql-join/magic-mallrats.sql

```diff
@@ -0,0 +1,6 @@
+select "c"."firstName",
+       "c"."lastName"
+from "films"
+join "inventory" using ("filmId")
+join "rentals" using ("inventoryId")
+join "customers" as "c" using ("customerId");
```

Comment:
Where does `'Magic Mallrats'` fit in?

---

![image](https://user-images.githubusercontent.com/16143771/118377493-5d434e00-b582-11eb-8486-23023f09b633.png)